packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -0,0 +1,238 @@
+import ResultError from './common/other/_ResultError';
+import NoData from './common/other/_KnownNoData';
+import styles from './ClinicalResult.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Result Object with the clinical result data.
+   */
+  resultData: PropTypes.shape({
+    /**
+     *  Blood Pressure grouped result id
+     */
+    id: PropTypes.string,
+    /**
+     *  Systolic Result for blood pressure.
+     */
+    systolic: observationPropShape,
+    /**
+     *  Diastolic Result for blood pressure.
+     */
+    diastolic: observationPropShape,
+  }),
+  /**
+   * Visually hides the unit of measure when presented in a series of side-by-side columns of the same unit.
+   */
+  hideUnit: PropTypes.bool,
+  /**
+   * Whether or not the text should be truncated in display. Restricts clinical result details each to one line.
+   */
+  isTruncated: PropTypes.bool,
+  /**
+   * Override that shows an Error display. Used when there is a known error or problem when retrieving or assembling the clinical result data.
+   */
+  hasResultError: PropTypes.bool,
+  /**
+   * Override that shows a known "No Data" display. Used when there is known to be no value for a given clinical result concept at a specific datetime.
+   */
+  hasResultNoData: PropTypes.bool,
+};
+
+const defaultProps = {
+  resultData: {},
+  hideUnit: false,
+  isTruncated: false,
+  hasResultError: false,
+  hasResultNoData: false,
+};
+
+const isEmpty = (str) => (!str || str.length === 0);
+
+const ClinicalResultBloodPressure = (props) => {
+  const {
+    resultData,
+    hideUnit,
+    isTruncated,
+    hasResultError,
+    hasResultNoData,
+    ...customProps
+  } = props;
+
+  let clinicalResultBloodPressureDisplay = null;
+
+  if (hasResultError || hasResultNoData) {
+    clinicalResultBloodPressureDisplay = hasResultError ? (<ResultError />) : (<NoData />);
+  } else {
+    const CompareTemplate = (s, d) => ({
+      systolic: s,
+      diastolic: d,
+    });
+
+    const compareUnits = new CompareTemplate(null, null);
+    const compareConceptDisplays = new CompareTemplate(null, null);
+    const compareDatetimeDisplays = new CompareTemplate(null, null);
+
+    let hasModifiedIcon = false;
+    let hasCommentIcon = false;
+    let hasUnverifiedIcon = false;
+
+    let iconGroupDisplayElement = null;
+    let conceptDisplayElement = null;
+    let datetimeDisplayElement = null;
+    const decoratedResultDisplay = [];
+
+    const hasSystolic = resultData.systolic;
+    const noDataSystolic = (hasSystolic && resultData.systolic.resultNoData === true);
+    if (hasSystolic && !noDataSystolic) {
+      const {
+        result = null,
+        conceptDisplay = null,
+        dateTimeDisplay = null,
+        isModified = null,
+        hasComment = null,
+        isUnverified = null,
+      } = hasSystolic;
+
+      if (!isEmpty(result.unit)) { compareUnits.systolic = result.unit.trim().toLowerCase(); }
```

Comment:
Generally I would put if's on just one line.
```suggestion
      if (!isEmpty(result.unit)) {
        compareUnits.systolic = result.unit.trim().toLowerCase();
      }
```